### PR TITLE
AQCU-1343: update lists from Aquarius 

### PR DESCRIPTION
**Pull request by mhines-usgs**
to include new LocationDatumPeriod as well as many others

### CODE REVIEW

**Comment by mhines-usgs** [comment]
Because the error in AQCU-1343 was reported from Aquarius in production, I ran the script @zmoore-usgs made against prod Aquarius.

File: src/main/resources/downloadSwagger.sh [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 #CONFIGURATION
-SWAGGERBASEURL="${1:-"http://tsqa.nwis.usgs.gov/AQUARIUS/Publish/v2"}"
+SWAGGERBASEURL="${1:-"http://ts.nwis.usgs.gov/AQUARIUS/Publish/v2/"}"
```
Comment (dpattermann-usgs) [inline src/main/resources/downloadSwagger.sh]: Do we want Dev/QA to point to ts.nwis ?

Comment (mhines-usgs) [reply src/main/resources/downloadSwagger.sh]: I'm not exactly sure how to manage this but maybe you guys already talked about this. The error I'm fixing was seen on prod, so I used that endpoint. But I imagine we may want to re-run this every time there is any kind of release made but would have to be careful not to release it to prod until that AQ version is also updated to whatever endpoint (QA?) we used to generate these files...

Comment (zmoore-usgs) [reply src/main/resources/downloadSwagger.sh]: Yeah, having the different versions across servers certainly makes it confusing. I'm thinking that from now on every time we run the script we probably want to tag the commit with whatever version of the Aquarius API it's run off of, and then if we need to build an older version later we can just checkout that tag.

**Pull request merged**